pages/content/amp-dev/page-experience.html

```diff
@@ -1,8 +1,9 @@
 ---
-$title: Page Experience
+$title: AMP Page Experience tool
 $view: /views/custom.j2
```

Comment:
Should we add `<meta name="robots" content="noindex">`?

Reply:
Hm... I'd advocate against. Setting pages to `noindex` made them undiscoverable for Google for a long time when we were launching amp.dev or was this unrelated?

Reply:
Why don't we want the page to be indexed?

Reply:
Given reindexing isn't deterministic - we don't want to be caught in that moment before launch where the page is queued for reindexing.

Reply:
Added the meta, PTAL

Reply:
I took @nainar's comment the other way, we want to ensure the page is indexed.

Reply:
I thought we don't want the "Coming soon" page to be indexed because we would prefer the post-launch page to be indexed.

Reply:
Yes sorry I meant we should have this indexed. Since it is the same URL, if we opt out of reindexing now and then opt in to it later, we can't guarantee that the page will be reindexed in time for launch. This means there is a risk that we wouldn't have the final page be reindexed/crawlable in time. 

To be clear: Please do not use `noindex`

Reply:
Thanks all for the additional points of clarification. Removed